src/goto-symex/path_storage.h

```diff
@@ -68,16 +68,8 @@ class path_storaget
   /// that check that the worklist is always empty when symex finishes.
   virtual void clear() = 0;
 
-  /// \brief Add paths to resume to the storage
-  ///
-  /// Symbolic execution is suspended when we reach a conditional goto
-  /// instruction with two successors. Thus, we always add a pair of paths to
-  /// the path storage.
-  ///
-  /// \param next_instruction: the instruction following the goto instruction
-  /// \param jump_target: the target instruction of the goto instruction
-  virtual void
-  push(const patht &next_instruction, const patht &jump_target) = 0;
+  /// \brief Add a path to resume to the storage
+  virtual void push(const patht &) = 0;
```

Comment:
I'd suggest to not make this one abstract - all current implementations are the same, so we might as well provide a default implementation. Future deriving classes are still free to override.

Reply:
They would also have to inherit unused internal storage.

Reply:
Ah, sorry for missing that.